src/components/main/classify/Classifier.js

```diff
@@ -0,0 +1,62 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { makeStyles } from '@material-ui/core/styles';
+// import MoodIcon from '@material-ui/icons/Mood';
+// import { useTranslation } from 'react-i18next';
+import './classifier.css';
```

Comment:
Actually, I was trying to implement an animation which seemed a lot more difficult to implement with makeStyles. We should probably discuss it. Otherwise, same as https://github.com/graasp/graasp-app-sticky-notes/pull/47#discussion_r845873123